<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Free CRM Application Test Automation Suite" duration-ms="43342" started-at="2020-06-21T09:33:54Z" finished-at="2020-06-21T09:34:37Z">
    <groups>
    </groups>
    <test name="Fee CRM Test Cases" duration-ms="43342" started-at="2020-06-21T09:33:54Z" finished-at="2020-06-21T09:34:37Z">
      <class name="com.crm.qa.tests.ContactPageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.crm.qa.tests.ContactPageTest@43738a82]" name="setUp" is-config="true" duration-ms="19033" started-at="2020-06-21T09:33:54Z" finished-at="2020-06-21T09:34:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="verifyContactlable()[pri:1, instance:com.crm.qa.tests.ContactPageTest@43738a82]" name="verifyContactlable" duration-ms="1545" started-at="2020-06-21T09:34:13Z" finished-at="2020-06-21T09:34:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Contact lable is missing in contact page expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Contact lable is missing in contact page expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at com.crm.qa.tests.ContactPageTest.verifyContactlable(ContactPageTest.java:57)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyContactlable -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.crm.qa.tests.ContactPageTest@43738a82]" name="tearDown" is-config="true" duration-ms="45" started-at="2020-06-21T09:34:14Z" finished-at="2020-06-21T09:34:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.crm.qa.tests.ContactPageTest@43738a82]" name="setUp" is-config="true" duration-ms="21271" started-at="2020-06-21T09:34:14Z" finished-at="2020-06-21T09:34:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="selectContactText()[pri:2, instance:com.crm.qa.tests.ContactPageTest@43738a82]" name="selectContactText" duration-ms="1379" started-at="2020-06-21T09:34:36Z" finished-at="2020-06-21T09:34:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- selectContactText -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.crm.qa.tests.ContactPageTest@43738a82]" name="tearDown" is-config="true" duration-ms="45" started-at="2020-06-21T09:34:37Z" finished-at="2020-06-21T09:34:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.crm.qa.tests.ContactPageTest -->
    </test> <!-- Fee CRM Test Cases -->
  </suite> <!-- Free CRM Application Test Automation Suite -->
</testng-results>
